src/test/java/uk/gov/companieshouse/pscdataapi/config/ExceptionHandlerConfigTest.java

```diff
@@ -0,0 +1,151 @@
+package uk.gov.companieshouse.pscdataapi.config;
+
+
```

Comment:
since there's an import comment below, perhaps the extra new line could be removed too